project/dexter-monitor/routes/server.js

```diff
@@ -155,6 +178,7 @@ function createEmailContentsWhenServerDown(server){
 
 function stopCheckingServerStatus(){
     clearInterval(checkingServerStatus);
+    checkingServerStatus = null;
```

Comment:
`doNotStartChecking` flag is just for unit testing, so it always has `undefined` value for production use. In unit testing, the flag is set to `true` not to call server heartbeat checking code.